african-marketplace/package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.4.3",
```

Comment:
Not sure that will be a good idea to add few UI libraries, but let's see what we will get